common-test/pom.xml

```diff
@@ -20,6 +20,12 @@
 			<artifactId>guava-testlib</artifactId>
 			<version>${guava.version}</version>
 		</dependency>
+		<dependency>
+			<!-- The stuff that java should have had from the start -->
```

Comment:
It's only used by tests. The common-test module should never be depended on by production code anyway, so guava will not follow along when jargo is packaged.